helpers.js

```diff
@@ -1,16 +1,4 @@
-import { DefaultTree } from '@neovici/cosmoz-tree/cosmoz-default-tree';
-import basicTree from './test/data/basicTree';
-import { tagged as css } from '@neovici/cosmoz-utils';
-
-const tree = new DefaultTree(basicTree),
```

Comment:
In many case we were using this `tree` instead of whatever was passed to `cosmoz-treenode-navigator`.
When we searched we were getting results which didnt even existed.